core/src/test/java/kafka/server/share/SharePartitionTest.java

```diff
@@ -1209,34 +1352,42 @@ public void testAcquireReleasedRecordMultipleBatches() {
         // Fourth fetch request with 5 records starting from offset 28.
         MemoryRecords records4 = memoryRecords(5, 28);
 
-        List<AcquiredRecords> acquiredRecordsList = sharePartition.acquire(
+        List<AcquiredRecords> acquiredRecordsList = fetchAcquiredRecords(sharePartition.acquire(
             MEMBER_ID,
+            Integer.MAX_VALUE,
             new FetchPartitionData(Errors.NONE, 40, 3, records1,
-                Optional.empty(), OptionalLong.empty(), Optional.empty(), OptionalInt.empty(), false));
+                Optional.empty(), OptionalLong.empty(), Optional.empty(), OptionalInt.empty(), false)),
+            5);
 
         assertArrayEquals(expectedAcquiredRecords(records1, 1).toArray(), acquiredRecordsList.toArray());
         assertEquals(15, sharePartition.nextFetchOffset());
 
-        acquiredRecordsList = sharePartition.acquire(
+        acquiredRecordsList = fetchAcquiredRecords(sharePartition.acquire(
             MEMBER_ID,
+            Integer.MAX_VALUE,
```

Comment:
Same comment as the 100 one, can we put Integer.MAX_VALUE as a variable for DEFAULT_MAX_FETCH_RECORDS

Reply:
Done.